src/components/global/Header/NavigationLinks/NavigationLinks.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Link, useLocation } from 'react-router-dom';
+import { routes } from '../../../../Router';
+import styles from '../Header.module.scss';
+
+export default function NavigationLinks() {
+    const location = useLocation();
+    const address = location.pathname;
+    const split_path = address.split('/');
+
+    const last_path = split_path[split_path.length - 1] !== '' 
+    ? split_path[split_path.length - 1] 
```

Comment:
@hubert-deriv please use option+shift+F here, it looks like this ternary expression can be aligned better 😊

Reply:
fixed